service/dap/server.go

```diff
@@ -2534,6 +2660,26 @@ func (s *Server) sendNotYetImplementedErrorResponse(request dap.Request) {
 		fmt.Sprintf("cannot process %q request", request.Command))
 }
 
+func (s *Server) sendOutputResponse(request dap.Request, category, message string) {
```

Comment:
This is confusing because Response has a very specific meaning in DAP and what you are sending is an event, which is not the same.

I think existing helper logToConsole should be sufficient for your purposes.